package.json

```diff
@@ -2,7 +2,7 @@
   "name": "panzoom",
   "version": "2.5.0",
   "description": "Pan and zoom SVG elements",
-  "main": "index.js",
+  "main": "dist/panzoom.min.js",
```

Comment:
Yes, but by putting `dist/panzoom.min.js` into `main` clause we are loosing opportunity for other bundlers to inline this module with proper source maps. We are also loosing benefits of improved minification algorithms, that new library clients can decide to use in future.